test/components/RecordAttributes_test.js

```diff
@@ -309,7 +309,7 @@ describe("RecordAttributes component", () => {
         });
 
         it("should show a custom field", () => {
-          expect(field.getAttribute("type")).to.eql("text");
+          expect(field.getAttribute("type")).to.eql("integer");
```

Comment:
This is wrong. The `ui:widget` for the `schema` field is overridden to be `text` instead of `integer`